unstoppable-ios-app/domains-manager-ios/Modules/LoginViewController/LoginViewPresenter.swift

```diff
@@ -92,8 +93,17 @@ private extension LoginViewPresenter {
     }
     
     @MainActor
-    func userAuthorized() {
-        view?.cNavigationController?.popViewController(animated: true)
+    func userAuthorized() async {
```

Comment:
It means `didUserAuthorize` or `isUserAuthorized`?

Reply:
Updated to userDidAuthorize()